frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -55,9 +56,10 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
 
   const namespace = match.params.ns || searchParams.get('preselected-ns');
   const initialChartURL = decodeURIComponent(searchParams.get('chartURL'));
+  const indexEntry = decodeURIComponent(searchParams.get('indexEntry'));
```

Comment:
This decodeURIComponent is not needed as URLSearchParams already understand, split and extract the query or search parameters from the URL.

```suggestion
  const indexEntry = searchParams.get('indexEntry');
```